app/middleware/tracking.js

```diff
@@ -0,0 +1,28 @@
+import AppConstants from 'constants/AppConstants';
+
+export const track = (trackData) => {
+  if (trackData) {
+    let interval = null;
+    const pushAndClear = () => {
```

Comment:
- [ ] Use function declaration here as well and [move the declaration out from the if block](https://github.com/airbnb/javascript#functions--in-blocks).